benchmarks/python/python_benchmark.py

```diff
@@ -289,7 +290,9 @@ async def main(
     if clients_to_run == "all" or clients_to_run == "glide":
         # Glide Socket
         client_class = GlideClusterClient if is_cluster else GlideClient
-        config = BaseClientConfiguration(
+        config = GlideClusterClientConfiguration(
```

Comment:
unrelated - please revert

Reply:
nevermind - this fixes the removal of the BaseClientConfiguration from our exports.  👍